numpy/f2py/crackfortran.py

```diff
@@ -1434,8 +1434,7 @@ def analyzeline(m, case, line):
         last_name = None
         for l in ll:
             l[0], l[1] = l[0].strip(), l[1].strip()
-            if l[0].startswith(','):
-                l[0] = l[0][1:]
+            l[0] = l[0].removeprefix(',')
```

Comment:
not sure if performance matters here, but if it does, then it would help to merge this into the line above

Reply:
I merged them anyway, the resulting code looks better to me.